updating/updating-cluster-cli.adoc

```diff
@@ -24,6 +24,10 @@ See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permis
 * If your cluster uses manually maintained credentials with the AWS Secure Token Service (STS), obtain a copy of the `ccoctl` utility from the release image being upgraded to and use it to process any updated credentials. For more information, see xref:../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#sts-mode-upgrading[Upgrading an OpenShift Container Platform cluster configured for manual mode with STS].
 * Ensure that you address all `Upgradeable=False` conditions so the cluster allows an upgrade to the next minor version. You can run the `oc adm upgrade` command for an output of all `Upgradeable=False` conditions and the condition reasoning to help you prepare for a minor version upgrade.
 
+[IMPORTANT]
```

Comment:
I think that the original location made more sense - it's closer to when an upgrade would actually be failing. Also, you really shouldn't put three important notes on top of each other - it decrease their importance and makes them less readable. If you _must_ keep this here, I suggest condensing all of the important notes into a single note. (And the bug for the final note is closed, so you can most likely remove it.)

Reply:
@wking or @LalatenduMohanty thoughts?

Reply:
I liked this location so folks who were expecting rollbacks to be supported were still able to reconsider things like taking etcd backups or whatever before launching themselves off through the one-way gate.  But yes, the old location is closer to where you'd be if you got stuck and were wondering "now what?".  I don't feel too strongly about it, if folks feel like they prefer the previous location, or have clever ideas to restructure to cover both use-cases.

Reply:
@wking thoughts on the new structure?